setup.py

```diff
@@ -237,102 +288,98 @@ def run(self):
         # Make sure the extension is built before the Python module is "built",
         # otherwise SWIG's generated module isn't included.
         # See https://stackoverflow.com/questions/12491328
-        self.run_command('build_ext')
+        self.run_command("build_ext")
         _build.run(self)
 
+
 class install(_install):
     def run(self):
         # See https://stackoverflow.com/questions/12491328
-        self.run_command('build_ext')
+        self.run_command("build_ext")
         _install.run(self)
 
+
 class bdist(_bdist):
     def finalize_options(self):
         _bdist.finalize_options(self)
         self.dist_dir = os.path.relpath(dist_dir)
 
+
 class bdist_wheel(_bdist_wheel):
     def run(self):
         if platform.system() == "Darwin":
-            os.environ['MACOSX_DEPLOYMENT_TARGET'] = '10.10'
+            os.environ["MACOSX_DEPLOYMENT_TARGET"] = "10.10"
```

Comment:
I assume your IDE or some tool has formatted automatically. I'm happy about it, but it would have been better in a separate commit, for ease of reviewing, because now it's difficult to see in the diff what is the actual addition of Python 3.11 and what is just the noise from reformatting.